### PR TITLE
Add adrelevantis adapter

### PR DESCRIPTION
**Pull request by ghguo**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by bretg** [comment]
Docs PR https://github.com/prebid/prebid.github.io/pull/2329

**Review by jsnellbaker — CHANGES_REQUESTED** [review]
Hi @ghguo 

I checked through the code and it largely seems good.  There are a few items that should be reviewed and addressed.

I tested the 3 different media types using the parameters in the md file and they rendered successfully.

I checked the test-coverage and it was just a bit below the accepted threshold.  It was at 74.41% and we'd like to see at least 80%.  Could you take a look at the unit tests and either add tests or modify the existing ones to improve the coverage?

I think we should good once the above is addressed.

Thanks.

File: modules/adrelevantisBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,616 @@
...
+      tag[NATIVE] = {layouts: [nativeRequest]};
+    }
+  }
+
+  const videoMediaType = utils.deepAccess(bid, `mediaTypes.${VIDEO}`);
+  const context = utils.deepAccess(bid, 'mediaTypes.video.context');
+
+  if (videoMediaType && context === 'adpod') {
+    tag.hb_source = 7;
+  } else {
```
[the remark this answers is not in the document]
Comment (ghguo) [reply modules/adrelevantisBidAdapter.js]: Hi jsnellbaker,

The suggested changes have been made. Please advise.

Thanks,
Gary

**Review by jsnellbaker — APPROVED** [review]
Thanks for making the updates; LGTM

**Pull request merged**